packages/twenty-front/src/generated-metadata/graphql.ts

```diff
@@ -1803,11 +1829,11 @@ export type Workspace = {
 };
 
 export enum WorkspaceActivationStatus {
-  ACTIVE = 'ACTIVE',
-  INACTIVE = 'INACTIVE',
-  ONGOING_CREATION = 'ONGOING_CREATION',
-  PENDING_CREATION = 'PENDING_CREATION',
-  SUSPENDED = 'SUSPENDED'
+  Active = 'ACTIVE',
```

Comment:
nope! Your PR might need to be rebased, looks outdated :)

Reply:
Fix in 4e1e43441642d43b9903eecada317e5fae787a42